website/docs/installation/get_started_on_kind.md

```diff
@@ -13,7 +13,7 @@ Before deployment, make sure [Docker](https://docs.docker.com/install/) is insta
 ## Install Chaos Mesh
 
 ```bash
-curl -sSL https://raw.githubusercontent.com/pingcap/chaos-mesh/master/install.sh | sh -s -- --local kind
+curl -sSL https://raw.githubusercontent.com/pingcap/chaos-mesh/master/install.sh | bash -s -- --local kind
```

Comment:
we also need to update this in `pages/index.js`

Reply:
Done.